instrumentation/liberty/liberty/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/liberty/ThreadLocalContext.java

```diff
@@ -45,6 +48,10 @@ public HttpServletRequest getRequest() {
     return request;
```

Comment:
and then can get rid of the `request` field
```suggestion
    return requestContext.request();
```